static/app/views/insights/common/utils/useSamplesDrawer.tsx

```diff
@@ -22,7 +22,7 @@ export function useSamplesDrawer({
   Component,
   moduleName,
   requiredParams,
-  onClose = () => undefined,
+  onClose = undefined,
```

Comment:
We can just remove the default if this is the case, the type already mentions that `onClose` is optional

Reply:
Oh true, thanks for pointing out! I'll add this improvement to the follow-up test-PR